app/app.go

```diff
@@ -172,6 +172,8 @@ func (a *App) Start(ctx context.Context) error {
 		grpc.WithRestProxyPort(a.cfg.GetInt(config.SpaceRestProxyServerPort, 0)),
 	)
 
+	textileClient.ListenForMessages(ctx, srv)
```

Comment:
Maybe this can be started in the healthcheck / start function of textile client

Reply:
For this I needed to pass in `srv` which is not currently exposed in the textile client (this is so that the message handler can issue a grpc stream event when a message is received). Also we start the grpc server after the textile client so `srv` isn't available when initing textile client :exploding_head: